src/test/java/com/hyperlocal/server/HyperlocalApplicationTests.java

```diff
@@ -0,0 +1,13 @@
+package com.hyperlocal.server;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+
+@SpringBootTest
+class HyperlocalApplicationTests {
+
+	@Test
```

Comment:
Add Tests

Reply:
Doing that now. I've switched to a different branch. I was having issues opening the pull request again